features.org

```diff
@@ -216,25 +245,25 @@ More on [[file:manual/Publishing.html][publishing]] (manual)
 
 [[https://orgmode.org/worg/org-tutorials/org-latex-export.html][Tutorial on publishing]] to $\LaTeX$ (Worg)
 
-* Extensibility
+* Emacs Extensible
   :PROPERTIES:
   :CUSTOM_ID: extensibiliy
   :HTML_CONTAINER_CLASS: side-fig
   :END:
+# Extremely extensible
```

Comment:
I think I might preffer this (Extremely extensible) to "Emacs Extensible" - given that I want this page to make sense to people who don't use Emacs